### PR TITLE
Add custom icon component into Button component

### PR DESCRIPTION
**Pull request by tamdao**

### CODE REVIEW

File: build/bundle.js [hunk 1]
```diff
@@ -1 +1,4671 @@
...
\ No newline at end of file
+!(function(e) {
```
Comment (binoy14) [inline build/bundle.js]: @tamdao Whats the reason to change this file? We will be using travis to build this.

**Comment by binoy14** [comment]
@tamdao Can you please update the test. You can run `npm run test -- -u` and it would be good to update the docs on how to use this as well as add a new test for it

File: build/bundle.js (hunk 1 above)
Comment (tamdao) [inline build/bundle.js]: Sorry. I don't know about that. So Do I need to reset it?

**Comment by tamdao** [comment]
@binoy14 Yeah. I will add the test and update document for that thia weekend

**Comment by binoy14** [comment]
@tamdao Thanks for the PR! 🙌

**Pull request merged**